src/renderer/components/+extensions/extensions.tsx

```diff
@@ -308,22 +303,77 @@ async function installOnDrop(files: File[]) {
   await requestInstalls(files.map(({ path }) => path));
 }
 
+async function requestInstallVersion(url: URLParse): Promise<string | null> {
```

Comment:
Would it be easy to trigger this from a protocol handler (for example `lens://app/extension/from-url/:url`)? If yes, should we add it?

Reply:
Could handle getting versions other than latest this way too, using a fragment for the version?

Reply:
I would use a query param but yes that would work.

Reply:
of course :)